p2p/peerManager/peerdb/datastore_test.go

```diff
@@ -116,3 +110,26 @@ func TestQuery(t *testing.T) {
 		require.Equal(t, key, "/"+peers[0].AddrInfo.ID.String())
 	})
 }
+
+func TestGetSize(t *testing.T) {
```

Comment:
Does this test only check size with single key? If so we should test more complex scenarios like adding multiple in parallel to make sure the size variable gets updated correctly.

Reply:
I've increased keys number to 5, create  and  update them two times in parallel now.